File: src/BukkitPE/event/entity/ProjectileLaunchEvent.php
<?php
namespace BukkitPE\event\entity;

use BukkitPE\entity\Projectile;
use BukkitPE\event\Cancellable;

class ProjectileLaunchEvent extends EntityEvent implements Cancellable{
	public static $handlerList = null;

	/**
	 * @param Projectile $entity
	 */
	public function __construct(Projectile $entity){
		$this->entity = $entity;

	}

	/**
	 * @return Projectile
	 */
	public function getEntity(){
		return $this->entity;
	}

}

File: src/main/java/net/BukkitPE/event/entity/ProjectileLaunchEvent.java
package net.BukkitPE.event.entity;

import net.BukkitPE.entity.projectile.EntityProjectile;
import net.BukkitPE.event.Cancellable;
import net.BukkitPE.event.HandlerList;

public class ProjectileLaunchEvent extends EntityEvent implements Cancellable {

    private static final HandlerList handlers = new HandlerList();

    public ProjectileLaunchEvent(EntityProjectile entity) {
        this.entity = entity;
    }

    public static HandlerList getHandlers() {
        return handlers;
    }

    public EntityProjectile getEntity() {
        return (EntityProjectile) this.entity;
    }
}
